llama-index-core/llama_index/core/llms/callbacks.py

```diff
@@ -209,6 +209,19 @@ def dummy_wrapper(_self: Any, *args: Any, **kwargs: Any) -> Any:
         if not is_wrapped:
             f.__wrapped__ = True  # type: ignore
 
+        for attr in (
```

Comment:
Can we add a comment for what this is doing? I actually don't quite get what is happening here haha

Reply:
Added. 

Screenshots below show the difference before/after the pass-through.

## before
<img width="969" alt="Screenshot 2024-05-19 at 11 17 50 PM" src="https://github.com/run-llama/llama_index/assets/80478925/e1474370-196b-4496-86e8-5fd23bbf2dae">

## after
<img width="715" alt="Screenshot 2024-05-19 at 11 16 04 PM" src="https://github.com/run-llama/llama_index/assets/80478925/b16f9a45-3552-4975-97d1-11d07a3c15b7">